### PR TITLE
Add new file Facebook container per bug 1453144

### PR DESCRIPTION
**Pull request by peiying2**
*PR https://github.com/mozilla/bedrock/pull/5547/files - FB container
*PR https://github.com/mozilla/bedrock/pull/5556/files - update manifesto title

### CODE REVIEW

**Comment by peiying2** [comment]
@flodolo want to get this out soon so Stephan can work on it on his Friday. I copied and pasted the strings from the PR.

**Comment by flodolo** [comment]
> @flodolo want to get this out soon so Stephan can work on it on his Friday. I copied and pasted the strings from the PR.

We've talked about this in the past: don't copy strings, import the patch and extract the string. If there's a variable, the string is wrong, and you need to test that strings are correctly extracted.

**Comment by flodolo** [comment]
> We've talked about this in the past: don't copy strings, import the patch and extract the string. If there's a variable, the string is wrong, and you need to test that strings are correctly extracted.

https://mozilla-l10n.github.io/documentation/products/mozilla_org/working_bedrock.html#try-a-patch-locally

**Comment by flodolo** [comment]
> @flodolo want to get this out soon so Stephan can work on it on his Friday. I copied and pasted the strings from the PR.

I just realized that we're talking about a pull request that was merged 10 days ago (Facebook Containers). What's going on?

File: en-US/firefox/facebookcontainer.lang [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+Watch the video
+
+
+;Opt out on your terms
+Opt out on your terms
+
+
+# The link points to https://addons.mozilla.org/firefox/addon/facebook-container/
+;Facebook can track almost all your web activity and tie it to your facebook identity. If that’s too much for you, the <a href="%(fbcontainer)s">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
+Facebook can track almost all your web activity and tie it to your facebook identity. If that’s too much for you, the <a href="%(fbcontainer)s">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en-US/firefox/facebookcontainer.lang]: Craig is working on it now.

File: en-US/firefox/facebookcontainer/index.lang [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Watch the video
+
+
+;Opt out on your terms
+Opt out on your terms
+
+
+# The link points to https://addons.mozilla.org/firefox/addon/facebook-container/
+;Facebook can track almost all your web activity and tie it to your Facebook identity. If that’s too much for you, the <a href="%(fbcontainer)s">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
+Facebook can track almost all your web activity and tie it to your Bacebook identity. If that’s too much for you, the <a href="%(fbcontainer)s">Facebook Container extension</a> isolates your identity into a separate container tab, making it harder for Facebook to track you on the web outside of Facebook.
```
Comment (flodolo) [inline en-US/firefox/facebookcontainer/index.lang]: Notice anything wrong? :-\

Also, you need for a developer to fix the original string before exposing it, and push it to production before the weekend. Have you checked with Craig? It shouldn't be a problem, but it need to be confirmed.

Comment (flodolo) [reply en-US/firefox/facebookcontainer/index.lang]: I see there's a PR, that's good. Still, there's a typo in your string.

File: en-US/firefox/facebookcontainer/index.lang [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+The Facebook Container Extension is not available on mobile devices.
+
+
+# GLink points to https://www.mozilla.org/firefox/focus/; Product name in German is Firefox Klar.
+;Try <strong>Firefox Focus</strong>, the privacy browser for Android and iOS.
+Try <strong>Firefox Focus</strong>, the privacy browser for Android and iOS.
+
+
+;Download on the App Store
+Download on the App Store
```
Comment (flodolo) [inline en-US/firefox/facebookcontainer/index.lang]: We already have this string in a shared file, that's why I didn't include in my comment.

Having said that, your call, you can keep it or remove it.

**Pull request merged**